dbx/utils/common.py

```diff
@@ -203,15 +217,18 @@ def get_all_environment_names(self) -> Any:
         return list(self._get_deployment_config().keys())
 
 
-def get_deployment_config(path: str) -> AbstractDeploymentConfig:
+def get_deployment_config(path: str, template_variables_file: Optional[str] = None) -> AbstractDeploymentConfig:
     ext = path.split(".").pop()
     if ext == "json":
         return JsonDeploymentConfig(path)
     elif ext in ["yml", "yaml"]:
         return YamlDeploymentConfig(path)
     elif ext == "j2":
         second_ext = path.split(".")[-2]
-        return Jinja2DeploymentConfig(path, second_ext)
+        if template_variables_file:
+            return Jinja2DeploymentConfig(path, second_ext, template_variables_file)
+        else:
+            return Jinja2DeploymentConfig(path, second_ext)
```

Comment:
It looks like line 228-231 could be condensed into`return Jinja2DeploymentConfig(path, second_ext, template_variables_file)` given `None` default values in `Jinja2DeploymentConfig` and `get_deployment_config`